pycheribuild/projects/cross/cheribsd.py

```diff
@@ -563,8 +564,9 @@ def buildworld_args(self) -> MakeOptions:
     def kernel_make_args_for_config(self, kernconf: str, extra_make_args) -> MakeOptions:
         self._setup_make_args()  # ensure make args are complete
         kernel_options = self.make_args.copy()
-        if self.compiling_for_mips(include_purecap=True):
-            # Don't build kernel modules for MIPS
+        if self.compiling_for_mips(include_purecap=True) and not self.build_tests:
+            # Don't build kernel modules for MIPS, except if we are building tests
+            # (DTrace tests require the 'dtrace_test' loadable module)
             kernel_options.set(NO_MODULES="yes")
```

Comment:
I think we shouldn't bother building modules by default.
Maybe set `MODULES_OVERRIDE=dtrace_test` (https://www.freebsd.org/doc/handbook/kernelconfig-building.html) `if self.build_tests` and otherwise keep the NO_MODULES

Reply:
I'll change it so that it can be compiled directly into the kernel, and not as a module